projects/wallet-framework-dotnet.md

```diff
@@ -0,0 +1,65 @@
+# Project Name
+Wallet Framework .NET (wallet-framework-dotnet)
+
+# Project Description
+The wallet-framework-dotnet is a framework designed for .NET, focusing on providing a multi-platform wallet framework. Initially a part of the Hyperledger Aries project (Aries Framework .NET), this initiative has now branched out to cater to a broader audience. The primary aim is to create a multiprotocol wallet framework enabling implementations of OpenID4VC and SD-JWT VC, in accordance to the European Identity Wallet initiative's objectives.
+
+Currently, the framework supports DidComm v1 and AnonCreds. There is an active intention to extend support for other promising protocols, notably DidComm v2, to ensure the framework remains at the forefront of digital identity solutions.
+
+Furthermore, the team is considering the development of SD-JWT credentials as a standalone library. This might transition into a separate project proposal in the future, underscoring the commitment to modular and reusable components in the digital identity space.
```

Comment:
Please note that we have two SD-JWT projects in the OpenWallet Foundation. One is in Python and the other in Kotlin. I am not sure if either of those projects would satisfy your need for a standalone library. It would be good to work closely together with these two projects.

Reply:
I would also support having native SD-JWT implementation in different technologies. It's good however to see an alignment between all these projects: For example, share the same test scenarios and exercise their code base over them.